### PR TITLE
feat:Styling app

### PR DESCRIPTION
**Pull request by taher-software**
In this branch, I've used the book store design in zeplin to style my application. So please review how much my application match the zeplin application and send me your feedback before to move forward the next step.
Looking forward your review.
Best,

### CODE REVIEW

File: src/components/Books/BooksList.js [hunk 1]
```diff
@@ -9,15 +9,26 @@ const BooksList = () => {
...
+              <p className="category">{category}</p>
+              <h2>{title}</h2>
+              <p className="author">{author}</p>
+              <div className="user-btns">
+                <button className="edit-button" type="submit">Comments</button>
+                <button className="edit-button" type="submit" onClick={() => handleRemove(id)}>Remove</button>
+                <button className="edit-button" type="submit">Edit</button>
+              </div>
+            </div>
           </li>
```
Comment (ArthurGC) [inline src/components/Books/BooksList.js]: - [ ] As the requirements show: 😉 

design-requirements

It is necessary to implement this part too for each card. (You don't need to add any functionality for those.)

design-requirements-2

File: src/components/Books/Form.js [hunk 2]
```diff
@@ -7,28 +7,34 @@ const AddBook = () => {
...
-      <div>
-        <input type="text" placeholder="Book title" />
-        <select defaultValue="Category">
-          <option disabled selected>Category</option>
+    <form className="form">
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <div className="form-icone">
+        <input className="input-book" type="text" placeholder="Book title" />
+        <select className="category-input">
+          <option disabled selected hidden>Category</option>
```
Comment (ArthurGC) [inline src/components/Books/Form.js:32]: - [ ] You want to consider fixing the `width` property of your  `ADD NEW BOOK` text to match the template. ♻️ 

design-requirements-3

**Review by ArthurGC — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
Hi  @Taher-web-dev 

Amazing job!
There are some issues that you still need to work on to go to the next project but you are almost there! 



## Good Points :+1: :
You used proper GitHub Flow :heavy_check_mark:
Your linter checks are all passing :heavy_check_mark:
Your PR messages are descriptive :heavy_check_mark:
Your commit messages are meaningful :heavy_check_mark:

## Aspects to improve ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Emmyn5600 — APPROVED** [review]
STATUS: APPROVED :+1: 

Hello, @Taher-web-dev  👋🏾 👋🏾 congratulations on implementing all the changes requested by the previous reviewer as well as the project requirements.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

**Pull request merged**